recipes/pytest-codspeed/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "pytest-codspeed" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest_codspeed-{{ version }}.tar.gz
+  sha256: 665003fc20117b64a98d16ffd1008f5bd6bf3b1e9af142b98c00abff7f626bbd
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling
+    - pip
+  run:
+    - python >=3.7
+    - cffi >=1.15.1,<1.16.dev0
```

Upper pin no longer needed
```suggestion
    - cffi >=1.15.1
```